kolibri_instant_schools_plugin/auth/api.py

```diff
@@ -37,26 +37,13 @@ def validate_username(self, value):
             _('An account already exists for this phone number. To add a new profile ' +
                 'under this account, you must first login. If you have forgotten your ' +
                 'password, you can reset it using the link on the login page.'))
-        return value
+        return create_new_username(value)
```

Comment:
It was as simple as this the whole time 😅

Reply:
OK that makes sense, so this is only for the initial sign up, not profile creation.